CONTRIBUTING.md

```diff
@@ -1,28 +1,71 @@
+
+SumUp employees can also reach out in our internal Slack channel #design-system-web, although we strive to keep all communication in the open.
+
+## Submitting a pull request
+
+We appreciate pull requests (PRs) for smaller changes and bug fixes. For larger changes, we encourage you to [submit an issue](https://github.com/sumup-oss/circuit-ui/issues/new) to collect feedback first. The normal workflow looks as follows:
+
+1. Let others know that you're working on an issue by leaving a comment or assigning it to yourself.
+2. Clone the repository (you might need to fork it first) and branch out from the latest `main` branch.
+3. Code, add, commit, and push your changes in your feature branch.
+4. Submit a pull request and make sure that the CI checks pass. Don't forget to add a [changeset](#changesets) and accept the [contributor license agreement](#contributor-license-agreement-cla).
+5. Collaborate with the codeowners/reviewers to merge your changes to `main`.
+
+### Contributor License Agreement (CLA)
+
+To start contributing to SumUp Open Source projects, please accept our [Contributor License Agreement](https://opensource.sumup.com/cla). Should you have any questions or concerns, please get in touch with [opensource@sumup.com](mailto:opensource@sumup.com).
+
+## Quick Start
+
+### Prerequisites
+
+- [Node.js v14](https://nodejs.org/)
+- [Yarn 1](https://classic.yarnpkg.com/en/docs/install)
+
+### Installation
+
+- Run `yarn` in the repository's root directory to install everything you need for development.
+- Run `yarn bootstrap` in the root directory to build and set up the packages.
+
+### Available Scripts
+
+All packages related to the design system are organized in this monorepo. Thanks to [`yarn workspaces`](https://classic.yarnpkg.com/en/docs/workspaces) packages that depend on each other always use the latest version. [`lerna`](https://lerna.js.org/) makes it possible to run scripts across all packages at the same time. The list of scripts below can be run in each package directory individually, or in the repository's root directory for all packages at once.
+
+- `yarn start` — will build packages on every change
+- `yarn build` — will build packages once
+- `yarn test` — will run unit tests on every change
+- `yarn lint` — will lint the code once
+- `yarn docs` — will run the [Storybook](https://storybook.js.org/) in development mode
```

Comment:
Would it be more useful to point to the Circuit Storybook here?

```suggestion
- `yarn docs` — will run the [Storybook](https://circuit.sumup.com) in development mode
```

Reply:
Hm, perhaps. Storybook wasn't mentioned before, so the link was intended for anyone who's not familiar with it. I suppose they can also figure that out from our Storybook.